serde/src/java/org/apache/hadoop/hive/serde2/lazybinary/LazyBinaryArray.java

```diff
@@ -135,9 +135,9 @@ private void parse() {
     // adjust arrays
     adjustArraySize(arraySize);
     // find out the null-bytes
-    int arryByteStart = start + vInt.length;
-    int nullByteCur = arryByteStart;
-    int nullByteEnd = arryByteStart + (arraySize + 7) / 8;
+    int aarryByteStart = start + vInt.length;
+    int nullByteCur = aarryByteStart;
+    int nullByteEnd = aarryByteStart + (arraySize + 7) / 8;
```

Comment:
```suggestion
    int arrayByteStart = start + vInt.length;
    int nullByteCur = arrayByteStart;
    int nullByteEnd = arrayByteStart + (arraySize + 7) / 8;
```

Reply:
Sigh, I tried to do this manually last n night, not sure how I managed to mangle it so badly.